src/main/java/erika/task/Todo.java

```diff
@@ -0,0 +1,17 @@
+package erika.task;
+
+public class Todo extends Task{
+    public Todo (String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[T][%s] %s", isDone ? "X":" ", description);

```

Comment:
Consider using the parent class's `toString` method to apply inheritance principle